### DISCUSSION TITLE
Need help - Slinger was working (post-N9), now isn't

### DISCUSSION THREAD

**Discussion by DVoorhees1957** [GerryDazoo/Slinger#414] [Q&A] [answered]
I have 2 Slingboxes in LA, a 350 and a 500. A few days prior to N9, I tested both boxes locally (that is, running slingbox_server.py and VLC on my Win7 laptop, on the same LAN as the Slingboxes), then I powered the 500 off - just in case Mr. Sling got nasty - and left the 350 powered on. I then spent several weeks at another house, during which time I added rules to my LA router to allow traffic from the router at my new location, but to block all other inbound and outbound traffic to and from the Slingboxes. Post N9, I tested remote access to the 350 and everything was still working.

I got back to LA a few days ago, and today powered the 500 on (the 350 had been powered on all along). The router rules blocking traffic to and from the Slingboxes were (and are) still in place. However, when I tried testing with the same configuration that worked pre-N9 (my laptop and the Slingboxes on the same LAN), I now get an error starting up Slinger - the same error with both the 350 and the 500. 

Here is the slingbox_server console output for the case of trying to access the 500 (trying to access the 350 gets the same error):
```
C:\Tools\Slinger>slingbox_server.py conf500.ini
Version : 3.08e Running on Windows-7-6.1.7601-SP1 pid= 1500
Using config file conf500.ini
Connection Manager Running on port 8080 with 4 max streams using URL slingbox.
BOXES [('sb500', 'sb500')]
Streamer Running:  4 conf500.ini sb500 sb500 8080 8388608
starting up on  port 8080 8388608
Checking for slingbox at ('192.168.1.102', 55206)
Building page for sb500
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
Error connecting to  ('192.168.1.102', 55206)
sb500 No valid slingbox ip info found in config.ini
Finding Slingbox on local network. My IP Info =  fe80::ad0f:20ce:ed98:61d4%25
Error binding socket to send broadcast [Errno 11004] getaddrinfo failed
Finding Slingbox on local network. My IP Info =  192.168.1.124
Slingbox Found 192.168.1.102 5206 " LA Sling500 " FinderID <removed>
Slingbox Found 192.168.1.104 5201 " Sling350 " FinderID <removed>
Finding Slingbox on local network. My IP Info =  169.254.131.214
Finding Slingbox on local network. My IP Info =  ::1
Error binding socket to send broadcast [Errno 11004] getaddrinfo failed
Finding Slingbox on local network. My IP Info =  127.0.0.1
Found more than one slingbox on the local network.
Please select the one you want to use and update the config.ini accordingly.

sb500 Giving up. Sorry..
```

And here are the contents of the conf500.ini file:
```
[SLINGBOXES]
sb500=sb500

[sb500]
sbtype="350/500/M1/M2"
password=<removed>
ipaddress=192.168.1.102
port=55206
name=sb500
Resolution=12
FrameRate=60
VideoBandwidth=2000
VideoSmoothness=63
VideoSource=1
maxstreams=2

[SERVER]
port=8080
maxremotestreams=4
enableremote=yes
```

My laptop can ping both Slingboxes, so that's not the problem. And just to rule out the possibility that the router rules I added were somehow interfering with Slinger talking to the Slingboxes, I removed the router from the internet, then disabled all of the traffic-blocking rules that I added just after N9, then retested. I still got the same error.

Any ideas what might be wrong?

**Comment by GerryDazoo** [comment, marked as answer]
Yes, you fat fingered the port number 
port=55206
should be port=5206
I guess that wasn't obvious from the logs. Sorry

**Reply by DVoorhees1957** [reply]
Thanks! I think I know what happened - when we found out that Mr. Sling had indeed done some nasty things right after N9, out of paranoia I made the ports for remote access 55201 and 55206 (mapped to 5201 and 5206, respectively, via some router rules)... and then I must have copied a portion of the "remote" INI file to the "local" INI file.

Anyway, thanks so much for your prompt assistance - and your eagle eyes!
